arangod/Aql/AqlItemBlock.cpp

```diff
@@ -491,24 +498,26 @@ SharedAqlItemBlockPtr AqlItemBlock::cloneDataAndMoveShadow() {
     for (size_t row = 0; row < numRows; row++) {
```

Comment:
Can we just iterate to numEffectiveRows here?
If I am not mistaken in all paths there will be no action if row < numEffectiveRows